lib/Doctrine/ruleset.xml

```diff
@@ -599,9 +599,6 @@
     </rule>
     <!-- Forbid strings in `"` unless necessary -->
     <rule ref="Squiz.Strings.DoubleQuoteUsage"/>
-    <rule ref="Squiz.Strings.DoubleQuoteUsage.ContainsVar">
-        <message>Variable "%s" not allowed in double quoted string; use sprintf() or concatenation instead</message>
-    </rule>
```

Comment:
The default error message is:

> Variable "%s" not allowed in double quoted string; use concatenation instead

No need to override that.

Reply:
Should the block be kept, or is it just for specifying `<message/>`?

Reply:
If was needed for overriding the message. If we don't override the message, the line directly above should be enough:

```XML
<rule ref="Squiz.Strings.DoubleQuoteUsage"/>
```